sdk/lusid/api/transaction_portfolios_api.py

```diff
@@ -169,10 +203,11 @@ def adjust_holdings_with_http_info(self, scope, code, effective_at, adjust_holdi
 
         # Authentication setting
         auth_settings = ['oauth2']  # noqa: E501
-
-        # set the LUSID header
-        header_params['X-LUSID-SDK-Language'] = 'Python'
-        header_params['X-LUSID-SDK-Version'] = '0.11.3042'
+        
+        response_types_map = {
+            200: "AdjustHolding",
+            400: "LusidValidationProblemDetails",
+        }
```

Comment:
This is interesting, in the case of a 400 response does this still throw an `ApiException`? Or does it now return a `LusidValidationProblemDetails` instead?

Reply:
N/A now using old generator